vm-virtio/src/console.rs

```diff
@@ -0,0 +1,353 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+use epoll;
+use libc::EFD_NONBLOCK;
+use std;
+use std::cmp;
+use std::collections::VecDeque;
+use std::io;
+use std::os::unix::io::AsRawFd;
+use std::result;
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::{Arc, Mutex};
+use std::thread;
+
+use super::Error as DeviceError;
+use super::{
+    ActivateError, ActivateResult, DeviceEventT, Queue, VirtioDevice, VirtioDeviceType,
+    INTERRUPT_STATUS_USED_RING, VIRTIO_F_VERSION_1,
+};
+use crate::VirtioInterrupt;
+use vm_memory::{Bytes, GuestMemoryMmap};
+use vmm_sys_util::EventFd;
+
+const QUEUE_SIZE: u16 = 256;
+const NUM_QUEUES: usize = 2;
+const QUEUE_SIZES: &[u16] = &[QUEUE_SIZE];
```

Comment:
`QUEUE_SIZES` needs to have two elements of 256 since we expect two distincts queues.

Reply:
Done by @sboeuf .